scripts/ci/jobs/eks_qa_e2e_tests.py

```
@@ -3,15 +3,155 @@
+    print("Attaching EBS CSI policy to IAM role...")
+    subprocess.run(
+        ["aws", "iam", "attach-role-policy",
+         "--role-name", role_name,
+         "--policy-arn", "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"],
+        check=True,
+    )
+    print("Policy attached successfully")
+
+    # Install the EBS CSI driver addon
+    print("Creating EBS CSI driver addon...")
+    subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)